### ISSUE TITLE
Sync Structure - Medienpool

### ISSUE THREAD

**Issue by cukabeka** [tbaddade/redaxo_sprog#39]
Habe oft dieselben Kategorien in Medienpool und Structure. Könnte SPROG das nicht zusätzlich auch syncen? Würde thematisch ja gut hier reinpassen..

**Comment by olien** [comment]
Und die Sprachen ignorieren??

**Comment by cukabeka** [comment]
Man könnte ja eine Sprache auswählen, die der "Master" für die Medienpool-Kategorien ist

**Comment by tbaddade** [comment]
Ich verstehe das Problem/Feature gar nicht. Was genau soll gesynct werden?

**Comment by cukabeka** [comment]
Die Struktur von den Artikel-Kategorien in die Medienpool-Kategorien. 

Zum Beispiel in der REX-Demo:
![image](https://user-images.githubusercontent.com/688274/44366725-ec080680-a4cd-11e8-9531-fd1ad965e869.png)


Oft sind dieselben Kategorien auch für die Bilder notwendig. Das nicht identisch im Medienpool nochmal händisch anzulegen würde die Sache deutlich vereinfachen.

**Comment by tbaddade** [comment]
D.h. das du ein Bild, welches mehrfach in unterschiedlichen Kategorien genutzt wird, auch mehrmals hochlädst, nur damit es in der „Strukturkategorie“ liegt?

**Comment by cukabeka** [comment]
Nein, nicht unbedingt. Ggf muss der Redakteur sich merken, wo es lag - oder es gibt die Möglichkeit, zusätzliche Kategorien anzulegen, die nicht in Sync mit der Struktur sind. 
In dem Beispiel von oben zB

- Artikel-Medien
-- Homepage
-- Struktur
-- Templates
[etc]
- Eigene Kategorie 1
-- Eigene Unterkategorie 1
-- Eigene Unterkategorie 2
- Eigene Kategorie 2

bzgl Löschen: Ja, "Keine Kategorie" wäre dann eine Möglichkeit. Oder wenn es gelöst wird wie eben beschrieben, dann evtl ein Fallback auf "Artikel-Medien"?
